src/Workspaces/Core/Portable/Formatting/Engine/TokenStream.cs

```diff
@@ -271,28 +269,50 @@ public int GetCurrentColumn(SyntaxToken token)
 
         public int GetCurrentColumn(TokenData tokenData)
         {
-            return GetColumn(tokenData, _getTriviaData);
+            return GetColumn(tokenData, new TriviaDataGetter(this));
         }
 
         public int GetOriginalColumn(SyntaxToken token)
         {
             var tokenWithIndex = this.GetTokenData(token);
-            return GetColumn(tokenWithIndex, _getOriginalTriviaData);
+            return GetColumn(tokenWithIndex, new OriginalTriviaDataGetter(this));
+        }
+
+        interface ITriviaDataGetter
+        {
+            TriviaData GetTriviaData(TokenData token1, TokenData token2);
+        }
+
+        struct TriviaDataGetter : ITriviaDataGetter
+        {
+            readonly TokenStream _tokenStream;
+            public TriviaDataGetter(TokenStream tokenStream) { _tokenStream = tokenStream; }
+
+            public TriviaData GetTriviaData(TokenData token1, TokenData token2) => _tokenStream.GetTriviaData(token1, token2);
+        }
+
+        struct OriginalTriviaDataGetter : ITriviaDataGetter
+        {
+            readonly TokenStream _tokenStream;
+            public OriginalTriviaDataGetter(TokenStream tokenStream) { _tokenStream = tokenStream; }
+
+            public TriviaData GetTriviaData(TokenData token1, TokenData token2) => _tokenStream.GetOriginalTriviaData(token1, token2);
         }
```

Comment:
can these br broken out into spearate partial files?

Reply:
please put all accessibility modifiers on all members (i.e. the structs, the interfaces, and the fields.)

Reply:
@CyrusNajmabadi not sure about breaking these into separate files. Really they are an implementation detail very specific to `TokenStream`

Reply:
Fixed the accessibility modifiers, but left both structs in the file. I don't think they should be separated. Please, reopen if disagree with the reasoning/consider it insufficient.

Reply:
hi @lostmsu impl details can be in separate files thanks to "partial types"  just write it this way:

```
Whatever.cs
-------
partial class Whatever
{
    // normal stuff.
}

---
Whatever.Inner.cs
partial class Whatever
{
    private class Inner
    {
    }
}
```